kani-compiler/src/codegen_cprover_gotoc/codegen/rvalue.rs

```diff
@@ -694,6 +694,14 @@ impl<'tcx> GotocCtx<'tcx> {
                         let meta = self.codegen_operand_stable(&operands[1]);
                         slice_fat_ptr(typ, data_cast, meta, &self.symbol_table)
                     }
+                    TyKind::RigidTy(RigidTy::Str) => {
+                        let pointee_goto_typ = Type::unsigned_int(8);
+                        // cast data to pointer with specified type
+                        let data_cast =
+                            data.cast_to(Type::Pointer { typ: Box::new(pointee_goto_typ.clone()) });
```

Comment:
Do we really need `clone()` here?

Reply:
My bad, I used to have a log statement after this one.

Reply:
```suggestion
                            data.cast_to(Type::Pointer { typ: Box::new(pointee_goto_typ) });
```